### PR TITLE
Publish button

### PR DESCRIPTION
**Pull request by lpil**
airbag

- [x] Publish button

### CODE REVIEW

File: app/containers/EditorHome/test.js [hunk 1]
```diff
@@ -1,15 +1,61 @@
...
     const { eventIDs } = mapStateToProps(state);
     expect(eventIDs.sort()).to.deep.equal(['44', '83']);
   });
+
+  it('sets events', () => {
+    const { events } = mapStateToProps(state);
+    expect(events).to.deep.equal(state.events);
+  });
+
+  it('sets events', () => {
```
Comment (zoeabryant) [inline app/containers/EditorHome/test.js]: it 'sets conference' ?

File: app/components/EditorHome/index.js [hunk 2]
```diff
@@ -41,11 +42,18 @@ const EditorHome = ({ eventIDs }) => (
         <EventLink id={newEventID(eventIDs)} />
       </li>
     </ul>
+    <button onClick={() => publish({ events, community, conference })}>
+      Publish
```
Comment (zoeabryant) [inline app/components/EditorHome/index.js]: Publish to react.london or another random s3 bucket?

File: app/containers/EditorHome/test.js (hunk 1 above)
Comment (lpil) [inline app/containers/EditorHome/test.js]: Thanks!

File: app/components/EditorHome/index.js (hunk 2 above)
Comment (lpil) [inline app/components/EditorHome/index.js]: Currently each instance of the app can only publish to one bucket/location/whatever. We need to design and implement a draft system.

**Pull request merged**